labs-ios-starter/FavoritesCollectionViewController.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  FavoritesCollectionViewController.swift
+//  labs-ios-starter
+//
+//  Created by Kelson Hartle on 1/25/21.
+//  Copyright © 2021 Spencer Curtis. All rights reserved.
+//
+
+import UIKit
+
+class FavoritesCollectionViewController: UIViewController {
+    @IBOutlet weak var collectionView: UICollectionView!
+    
+    let cities = ["New York", "Rio", "Orlando", "Miami", "San Diego", "Houston", "Kansas City"]
+    
+    // MARK: - Life Cycle
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        collectionView.delegate = self
+        collectionView.dataSource = self
+        view.backgroundColor = .lightGray
+        collectionView.backgroundColor = .lightGray
+    }
+    
+
+    // MARK: - Navigation
+
+    // In a storyboard-based application, you will often want to do a little preparation before navigation
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        // Get the new view controller using segue.destination.
+        // Pass the selected object to the new view controller.
+    }
+
+}
+
+// MARK: - UICollectionViewDelegateFlowLayout, UICollectionViewDataSource
+extension FavoritesCollectionViewController: UICollectionViewDelegateFlowLayout, UICollectionViewDataSource {
+    
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        return cities.count
+    }
+    
+    func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        
+        let cell = collectionView.dequeueReusableCell(withReuseIdentifier: "Cell", for: indexPath) as! FavoriteCollectionViewCell
+        
+        //cell.layer.borderWidth = 2
+        //cell.layer.borderColor = UIColor.blue.cgColor
```

Comment:
Remove commented code before merging to `main`

Reply:
Forgot about this code thanks!